tower-http/src/request_id.rs

```diff
@@ -181,6 +181,7 @@ use tower_layer::Layer;
 use tower_service::Service;
 use uuid::Uuid;
 
+pub(crate) const REQUEST_ID: HeaderName = HeaderName::from_static("x-request-id");
```

Comment:
This still has the `x-` 😄 
But also, do you have a link to docs for the non-`x-`-prefixed header?

Reply:
Fixed, and rebased to resolve the conflict.

Reply:
https://www.rfc-editor.org/rfc/rfc6648 deprecates "X-" prefix header, and `request-id` is also commonly used same as `x-request-id`.

Reply:
It says "deprecates the convention for *newly defined* parameters" (emphasis mine) though. Do you have a source for the claim that it's commonly used?